cronos-play/getting-started_unreal/quick-start/wallet.md

```diff
@@ -0,0 +1,129 @@
+# Wallet
+
```

Comment:
I'd suggest mentioning that most of the functionality is insecure / meant for development testing, and that one should use the walletconect (Iink to it) instead

Reply:
Done